### PR TITLE
Add ExecutionResources multiplication

### PR DESCRIPTION
**Pull request by pefontana**
# Add ExecutionResources multiplication

We need ` ExecutionResources * u64` implementation in starknet_in_rust
https://github.com/starkware-libs/cairo-lang/blob/master/src/starkware/starknet/business_logic/execution/os_usage.py#L37

## Checklist
- [ ] Linked to Github Issue
- [x] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Review by fmoletta — APPROVED** [review]
LGTM ser

File: src/vm/runners/cairo_runner.rs [hunk 1]
```diff
@@ -1166,8 +1166,8 @@ impl Mul<usize> for ExecutionResources {
     fn mul(self, rhs: usize) -> ExecutionResources {
         let mut total_builtin_instance_counter = self.builtin_instance_counter.clone();
 
-        for (_, val) in total_builtin_instance_counter.iter_mut() {
-            *val *= rhs;
+        for (_buildin_name, counter) in total_builtin_instance_counter.iter_mut() {
```
Comment (fmoletta) [inline src/vm/runners/cairo_runner.rs:1169]: typo

Comment (pefontana) [reply src/vm/runners/cairo_runner.rs]: Thanks fede!
done

**Comment by pefontana** [comment]
> LGTM ser

A pleasure to comeback

**Review by Oppen — APPROVED** [review]

**Pull request merged**